types/row_proof.go

```diff
@@ -16,9 +16,15 @@ type RowProof struct {
 	RowRoots []tmbytes.HexBytes `json:"row_roots"`
 	// Proofs is a list of Merkle proofs where each proof proves that a row
 	// exists in a Merkle tree with a given data root.
-	Proofs   []*merkle.Proof `json:"proofs"`
-	StartRow uint32          `json:"start_row"`
-	EndRow   uint32          `json:"end_row"`
+	Proofs []*merkle.Proof `json:"proofs"`
+	// StartRow the index of the start row.
+	// Note: it's currently not used for verification, only for validation.
+	// If it's used downstream, make sure to validate it.
+	StartRow uint32 `json:"start_row"`
```

Comment:
[question] can we please clarify "it" here? Is this correct?

```suggestion
	// Note: StartRow is currently not used for verification, only for validation.
	// Before using a RowProof, users are expected to call Validate().
	StartRow uint32 `json:"start_row"`
```

If not, can we clarify this comment in another way?

Reply:
[4f4c617](https://github.com/celestiaorg/celestia-core/pull/1473/commits/4f4c6177a488ac4246ec7c10a04a922c8293bafd) ?